lib/active_admin/orm/active_record/comments.rb

```diff
@@ -5,9 +5,9 @@
 
 # Add the comments configuration
 ActiveAdmin::Application.inheritable_setting :comments,                   true
-ActiveAdmin::Application.inheritable_setting :show_comments_in_menu,      true
 ActiveAdmin::Application.inheritable_setting :comments_registration_name, 'Comment'
 ActiveAdmin::Application.inheritable_setting :comments_order,             "created_at ASC"
+ActiveAdmin::Application.inheritable_setting :comments_menu,              false
```

Comment:
I've updated the commit so that the default is `{}` - if you prefer handling omments_menu `true`, then I can update the logic to do that as well.